README.md

```diff
@@ -1,36 +1,45 @@
-Make some interesting changes to the code
-Push the new code to your own repository
-Make a pull request on the GitHub website
+<li>Fork the repository (on the GitHub page)</li>
+<li>Clone the code from your own GitHub repository (the fork)</li>
+<li>Make some interesting changes to the code</li>
+<li>Push the new code to your own repository</li>
+<li>Make a pull request on the GitHub website</li>
 The Mother now needs to accept the changes on the website too
-# PART TWO - BRANCES:
+
+### PART TWO - BRANCES:
 Now The Mother needs to invite all the other students as collaborators to the repository
 The students will then receive an email where they need to accept the collaboration
 The students will then need to do the following:
-Clone the mothers repository (not the fork) to their own computer
-Make a new branch in the recently cloned repository
-Make sure that you are working on the new branch
-Make some changes to the code
-Switch to master (or main) branch
-Observe that your changes are not there anymore
-Merge the new branch into master/main
-Push the new code to GitHub
-Observe the new code in GitHub
-Now make a new branch again and make sure you are working on it
-Make some changes to the code again.
-Push the code by using git push origin xxxxxx where xxxxxx is the name of your branch
+<li>Clone the mothers repository (not the fork) to their own computer</li>
+<li>Make a new branch in the recently cloned repository</li>
+<li>Make sure that you are working on the new branch</li>
+<li>Make some changes to the code</li>
+<li>Switch to master (or main) branch</li>
+<li>Observe that your changes are not there anymore</li>
+<li>Merge the new branch into master/main</li>
+<li>Push the new code to GitHub</li>
+<li>Observe the new code in GitHub</li>
+<li>Now make a new branch again and make sure you are working on it</li>
+<li>Make some changes to the code again.</li>
+<li>Push the code by using git push origin xxxxxx where xxxxxx is the name of your branch</li>
 The Mother now needs to do the following:
-Pull the new changes and check out the new branches from the other students
-Observe that the changes are there each time you check out a new branch
-Check out the master/main branch
-Merge the other branches into master/main
-Push the code to GitHub
\ No newline at end of file
+<li>Pull the new changes and check out the new branches from the other students</li>
+<li>Observe that the changes are there each time you check out a new branch</li>
+<li>Check out the master/main branch</li>
+<li>Merge the other branches into master/main</li>
+<li>Push the code to GitHub</li>
+
+## The Process 
+
+I did this project with my girlfriends who has some experiance with GitHub so I got to take on the both roles and she helped me out and explained everything to me. We took our time doing this project and went over everything in detail. I used <strong>GitKraken</strong> which is a really simple Git GUI software.
```

Comment:
Fix suggestions:

- girlfriends -> girlfriend
- I got to take on the both roles... -> I got to take on both roles (drop THE)
- Mention that you also did a pull request when branching as a collaborator